components/wasm-opt/src/base.rs

```diff
@@ -387,6 +416,7 @@ pub fn check_pass_options_defaults_os(pass_options: PassOptions) -> bool {
 }
 
 // FIXME binaryen unicode path handling is broken on windows
+// TODO: How do I fix this?
```

Comment:
I saw a [comment on the issue about issues with Windows paths](https://github.com/brson/wasm-opt-rs/issues/40#issuecomment-1776043915) saying this is fixed in binaryen now, but I'm not sure if any more work is needed here or if this repo gets it "for free".